### PR TITLE
Feature/ubi 245/gke deployment

### PR DESCRIPTION
**Pull request by MaximeStOnge**
#### What changes does this PR contain?
Resources override and Readme

#### Which JIRA ticket ID's are associated with this PR?
UBI-245

### CODE REVIEW

File: chart/values.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+alertmanager:
+  resources: 
+    limits:
```
Comment (ysma500) [inline chart/values.yaml:3]: Req first

File: chart/values.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      cpu: 10m
+      memory: 32Mi
+    requests:
+      cpu: 10m
+      memory: 32Mi
+
+nodeExporter:
+  resources: 
+    limits:
```
Comment (ysma500) [inline chart/values.yaml:12]: Req first

File: chart/values.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      cpu: 100m #default 200m
+      memory: 50Mi
+    requests:
+      cpu: 25m #default 100m
+      memory: 30Mi
+
+server:
+  resources: 
+      limits:
```
Comment (ysma500) [inline chart/values.yaml:21]: Req first

File: chart/values.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        cpu: 200m #default 500m
+        memory: 512Mi
+      requests:
+        cpu: 100m #default 500m
+        memory: 512Mi
+
+pushgateway:
+  resources: 
+    limits:
```
Comment (ysma500) [inline chart/values.yaml:30]: Req first

File: chart/values.yaml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+      cpu: 10m
+      memory: 32Mi
+    requests:
+      cpu: 10m
+      memory: 32Mi
```
Comment (ysma500) [inline chart/values.yaml:35]: New line EOF missing

**Review by ysma500 — CHANGES_REQUESTED** [review]

**Review by ysma500 — APPROVED** [review]

**Review by ArcticWalrus — APPROVED** [review]

**Pull request merged**